app/views/candidate_mailer/application_rejected_all_applications_rejected.text.erb

```diff
@@ -20,6 +20,9 @@ Contact <%= @course.provider.name %> directly if you have any questions about th
   You’ve now had decisions about all the courses you applied for. You did not get any offers, so you can apply again.
 <% end %>
 
-Your last application has been saved. You can make changes before you submit your new application.
+If now’s the right time for you, you can apply for teacher training again this year. 
```

Comment:
Other than the title this is all shared with the `_interested` partial. Is it worth pulling it out further? (The answer might be no, just wanted to ask the question).

Reply:
Fair point. Have refactored a bit DRY things a bit.